src/MockClassifier.Api/Services/Dmr/DmrRequest.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace MockClassifier.Api.Services.Dmr
+{
+    /**
+     * NOTE:
+     * These models live here temporarily until they can be referenced via a NuGet package library
+     */
+
+    /// <summary>
+    /// The model for the DMR API endpoint
+    /// </summary>
+    public record DmrRequest
+    {
+        /// <summary>
+        /// The destination URI for the <see cref="Payload"/>
+        /// </summary>
+        public string ForwardUri { get; set; }
+
+        /// <summary>
+        /// The payload being sent to the <see cref="ForwardUri"/>
+        /// </summary>
+        public Payload Payload { get; set; }
+    }
+
+    /// <summary>
+    /// The payload that the DMR handles
+    /// </summary>
+    public record Payload
```

Comment:
Since these models are living here temporarily, I figured it would be easier to keep them in the same file as they belong together and will be extracted to another library together also.

Happy to break it apart if you prefer though? Please let me know @guypritchard